  INFO root [2019-10-08 09:14:52,344] - 
  INFO root [2019-10-08 09:14:52,344] - -----------------------------------------------------------------------------------------


  INFO root [2019-10-08 09:14:52,344] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-10-08 09:14:52,344] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/HappyPoker_assets/;D:/kbengine/HappyPoker_assets/scripts/;D:/kbengine/HappyPoker_assets/res/
  INFO root [2019-10-08 09:14:52,344] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-10-08 09:14:52,406] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.43.114 ( 192.168.43.114:20013 )
  INFO root [2019-10-08 09:14:52,407] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.43.114:20013, SOMAXCONN=128.
  WARN root [2019-10-08 09:14:54,679] - NetworkInterface::initialize(INTERNAL-TCP): Couldn't parse interface spec 'ip_internal' so using all interfaces
  INFO root [2019-10-08 09:14:54,683] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.43.114 ( 192.168.43.114:53324 )
  INFO root [2019-10-08 09:14:54,684] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.43.114:53324, SOMAXCONN=128.
  INFO root [2019-10-08 09:14:54,684] - server-configs:
  INFO root [2019-10-08 09:14:54,684] - 	internalTcpAddr : 192.168.43.114:53324
  INFO root [2019-10-08 09:14:54,684] - 	externalTcpAddr : 192.168.43.114:20013
  INFO root [2019-10-08 09:14:54,684] - 	componentID : 9000
  WARN root [2019-10-08 09:14:54,843] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-10-08 09:14:54,947] - Components::findLogger: found logger, addr:192.168.43.114:53267
  INFO root [2019-10-08 09:14:54,947] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-10-08 09:14:54,947] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-10-08 09:14:54,953] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 41181. PID: 3628 ----
  INFO root [2019-10-08 09:14:54,954] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-10-08 09:14:54,984] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-10-08 09:14:54,984] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-10-08 09:14:55,243] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-10-08 09:14:55,243] - Script::install(): KBExtra is successfully!
S_INFO root [2019-10-08 09:14:55,264] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-10-08 09:14:55,264] - TelnetServer server is running on port 31000
  INFO root [2019-10-08 09:14:55,693] - ---- loginapp is running ----
 DEBUG root [2019-10-08 09:14:55,693] - Components::process(): Request for the process of identity...
  WARN root [2019-10-08 09:15:01,964] - Forwarding logs to logger[192.168.43.114:53267/0/0/0]...
 ERROR root [2019-10-08 09:18:09,658] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=584232173638, lastReceivedTime=584016553199, inactivityExceptionPeriod=599500512)
